javascript/net/grpc/web/grpc_generator.cc

```diff
@@ -688,9 +688,12 @@ void PrintProtoDtsMessage(Printer *printer, const Descriptor *desc, const FileDe
     printer->Print(vars, "set$js_field_name$(a: $js_field_type$): void;\n");
   }
   printer->Print(vars,
-                "toObject(): $class_name$.AsObject;\n"
                 "serializeBinary(): Uint8Array;\n"
-                "static deserializeBinary: (bytes: {}) => $class_name$;\n");
+                "toObject(includeInstance?: boolean): $class_name$.AsObject;\n"
+                "static toObject(includeInstance: boolean, msg: $class_name$): $class_name$.AsObject;\n"
+                "static serializeBinaryToWriter(message: $class_name$, writer: jspb.BinaryWriter): void;\n"
+                "static deserializeBinary(bytes: Uint8Array): $class_name$;\n"
+                "static deserializeBiinaryFromReader(message: $class_name$, reader: jspb.BinaryReader): $class_name$;\n");
```

Comment:
is that a typo?

```diff
- static deserializeBiinaryFromReader
+ static deserializeBinaryFromReader

Reply:
Looks like it, could you make PR?

Reply:
I fixed it directly on master. Thanks for the catch.